zuul-core/src/main/java/com/netflix/zuul/netty/timeouts/OriginTimeoutManager.java

```diff
@@ -124,4 +99,14 @@ private Long parseReadTimeoutMs(Object p) {
             return null;
         }
     }
+
+    @Nullable
+    private Object getRequestReadTimeout(HttpRequestMessage zuulRequest, Long defaultTimeout) {
+        return getRequestClientConfig(zuulRequest).getProperty(ReadTimeout, defaultTimeout);
```

Comment:
I know it's not your fault, but `ReadTimeout`, that is not good.

Reply:
Can you elaborate?

Reply:
The camelCasing is off for this.

Reply:
It's a statically imported constant, I can use the full class if you think that's clearer.